src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -249,10 +259,11 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
 
     # Trigger PPA creation.
     test_harness_ppa_geometry = ppa.PpaCreationModel(
-        config['registrationRequests'], pal_records)
+        config['registrationRequests'], config['palRecords'])
 
     # Check if the PPA generated by the SAS UUT is fully contained within the service area.
```

Comment:
For debugging purposes, we may want to log the SAS UUT PPA as well as the reference model's PPA. If this is already done implicitly, that's OK.

Reply:
Currently   PPA reference model doesn’t log debug message about PPA      so corresponding debug message are added here.